arithmetization/src/main/java/net/consensys/linea/zktracer/runtime/stack/StackLine.java

```diff
@@ -65,7 +67,7 @@ public StackLine() {
   public List<StackOperation> asStackOperations() {
     StackOperation[] r =
         new StackOperation[] {
-          new StackOperation(), new StackOperation(), new StackOperation(), new StackOperation()
+          EmptyStackOperation, EmptyStackOperation, EmptyStackOperation, EmptyStackOperation
```

Comment:
We can make it even more clear by keeping the StackOperation by exposing empty() as a factory method like for push and pop, which will always return the static emptyStackOperation.
```
StackOperation emptyStackOperation = StackOperation.empty();
StackOperation[] r = new StackOperation[] {
        emptyStackOperation  , emptyStackOperation, emptyStackOperation, emptyStackOperation};
```

Reply:
Should be addressed now.